src/FHIRConverterAPI/ConverterLogicHandler.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Microsoft.Health.Fhir.Liquid.Converter;
+using Microsoft.Health.Fhir.Liquid.Converter.Models;
+using Microsoft.Health.Fhir.Liquid.Converter.Models.Hl7v2;
+using Microsoft.Health.Fhir.Liquid.Converter.Processors;
+using Newtonsoft.Json;
+
+public static class ConverterLogicHandler
+{
+    private const string MetadataFileName = "metadata.json";
+    private static readonly ProcessorSettings DefaultProcessorSettings = new ProcessorSettings();
+
+    public static string Convert(string templateDirectory, string rootTemplate, string inputDataContent, bool isVerboseEnabled, bool isTraceInfo)
```

Comment:
how is this file different from the one in the converter itself? do we need both?

Reply:
could we consolidate those some so the processor returns a string and then there's another function to write it to a file for the cli? having this code duplicated makes me a bit nervous about drift